src/app/[locale]/incident/add/components/MapDialog.tsx

```diff
@@ -1,6 +1,16 @@
 import * as Dialog from '@radix-ui/react-dialog'
-import React, { useEffect, useMemo, useRef, useState } from 'react'
-import { uniqBy } from 'lodash'
+import React, {
```

Comment:
Volgende week bespreken:
@justiandevs 
waarom staat:
`scrollZoom={false}`

nu kan je niet meer scrollen op de kaart :(

Reply:
Ik zou dit even veranderen, ik heb dit gedaan omdat je dus op mobiel de kaart onder de lijst met addressen krijgt. Wil je weer naar boven kunnen scrollen dan is dit benodigd om ervoor te zorgen dat de scroll focus niet naar de kaart gaat maar naar de webpagina. 

Voor nu ga ik ervoor zorgen dat scrollZoom alleen disabled is op een mobiel device.